guidelines/REUSE.md

```diff
@@ -133,8 +209,20 @@ _Assuming file https://my.company.com/definitions/models.json_
 ```
 
 
+### Definitions
+
+Reuse schema definitions by creating a repository of definitions.  This is done by simply hosting a file or set of files for commonly used definitions across a company or organization.
+
+Refer to [External references](#external-references) for referencing strategies.
```

Comment:
Intentionally left this section pretty empty, as `#external-references` has so many definition/schema examples.